internal/gwsidecar/gwsidecar.go

```diff
@@ -51,16 +51,27 @@ type GwConnectionContext struct {
 	RemoteSliceGwNsmSubnet string
 }
 
+type WorkerClientRpc interface {
+	GetStatus(ctx context.Context, serverAddr string) (*GwStatus, error)
+}
+
+type workerClientConfig struct {
+	Client sidecar.GwSidecarServiceClient
```

Comment:
Can we rename `workerClientConfig` to something else which implies that this is a client to communicate with gw sidecar?

Reply:
done